src/transaction/log_page_buffer.c

```diff
@@ -7323,6 +7323,9 @@ logpb_checkpoint_trantable (THREAD_ENTRY * const thread_p)
       return ER_FAILED;
     }
 
+  // *INDENT-OFF*
+  log_lsa trantable_checkpoint_lsa { NULL_LSA };
+  // *INDENT-ON*
```

Comment:
Doing an assignment instead of initializer avoids disabled indentation.
```suggestion
  log_lsa trantable_checkpoint_lsa = NULL_LSA;
```